website/docs/components/app-footer/partials/code/component-api.md

```diff
@@ -0,0 +1,121 @@
+## Component API
+
+### AppFooter
+
+The base `AppFooter` component includes a copyright notice. It also wraps and contains the `AppFooter::LegalLinks`, `AppFooter::StatusLink`, `AppFooter::Link`, and `AppFooter::Item` child components as well as `ExtraBefore` and `ExtraAfter` yielded components.
+
+<Doc::ComponentApi as |C|>
```

Comment:
`AF.StatusLink` and `AF.LegalLinks` are missing (and use always the same order, the one in which they appear in the UI, and is recommended in the "Guidance" section)